spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -238,55 +252,56 @@
   end
 
   describe '#destroy' do
-    let(:work_to_be_deleted) { create(:private_generic_work, user: user) }
```

Comment:
Why rename the variable here? Do you want this `work` to stand in for the `work` created on line 7?